pynxtools/dataconverter/readers/em/examples/ebsd_database.py

```diff
@@ -0,0 +1,320 @@
+#
+# Copyright The NOMAD Authors.
+#
+# This file is part of NOMAD. See https://nomad-lab.eu for further info.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""Pieces of information relevant for the parsing the ebsd_database use case."""
+
+# for the Summer2023 EBSD database example with NOMAD OASIS:
+# we collected several EBSD datasets from colleagues. In these research studies,
+# names of the phases used for indexing (NXem_ebsd_crystal_structure_candidate)
+# instances were very different elements, substances, free-text annotations,
+# mineral names, names of mineral groups, or names of rocks
+# here is the FOR NOW human expert dictionary composed from these names
+# to map used phase names on atom types
+# the idea behind this was to identify what is the minimal information that
+# is recoverable when there is no common agreement about the phases used and their
+# exact atomic configuration
```

Comment:
Add this to the class docstring if you like. Those can end up in html docs if we need. These won't.

Reply:
did so in pynxtools-em, also in those places commented here